docs/cellxgene_census_docsite_landing.md

```diff
@@ -1,4 +1,4 @@
-🚀 <span style="background-color: PaleGoldenRod; color; font-size: 18px">  New memory-efficient features to get **gene expression average & variance**, and **highly variable genes.** Find out more [here.](articles/2023/20230919-out_of_core_methods.md)</span>
+<span style="background-color: #F6DDCC; color; font-size: 18px"> 🚨 A **normalized layer** and useful cell and gene **pre-calculated stats** are now available!  Find out more [here.](articles/2023/20231012-normalized_layer_precalc_stats.md)</span>
```

Comment:
Suggest slight rewording:    "... layer, plus useful cell and gene ...

Reply:
+1, and maybe also remove "useful" (why else we would add these, if not useful? 😄)